gatsby-node.ts

```diff
@@ -0,0 +1,28 @@
+import express, { Express } from "express";
+import type { GatsbyNode } from 'gatsby';
+import client from "./tina/__generated__/client";
+import path from "path";
+
+export const createPages: GatsbyNode['createPages'] = async ({ actions }) => {
+
+  const { createPage } = actions
+
+  const postsResult = await client.queries.postConnection({ first: 1000 });
```

Comment:
Do we need to specify the first 1,000 ? 

Can we leave this blank?

Reply:
✅Fixed

> Do we need to specify the first 1,000 ?

Nope! I supposed we don't need this for the amount of data in the starter 😬